editor/src/clj/editor/atlas.clj

```diff
@@ -217,7 +217,12 @@
   (inherits outline/OutlineNode)
 
   (property id g/Str
-            (value (g/fnk [maybe-image-resource] (some-> maybe-image-resource resource/proj-path path->id)))
+            (value (g/fnk [maybe-image-resource rename-patterns] 
+                          (let [rename-patterns (if (nil? rename-patterns) "" rename-patterns)
+                                id (some-> maybe-image-resource resource/proj-path path->id)]
+                            (if (nil? id)
+                              ""
```

Comment:
The old code would produce `nil` if the `(some-> maybe-image-resource ...)` expression returned `nil`. Perhaps we shouldn't change it?